apisix/plugins/grpc-web.lua

```diff
@@ -110,16 +150,17 @@ function _M.access(conf, ctx)
 
     -- set grpc body
     local body, err = core.request.get_body()
-    if err then
+    if err or not body then
         core.log.error("failed to read request body, err: ", err)
-        return 400
+        return exit(ctx, 400)
     end
 
     if encoding == CONTENT_ENCODING_BASE64 then
         body = decode_base64(body)
+        ngx.log(ngx.WARN, "DECODE BODY: ", body)
```

Comment:
1. why do we need warn log here? 
2. we can use core.log.warn instead

Reply:
Obviously here leaves out some unneeded code, which I will remove in another PR. 🫡

Reply:
This is probably code for testing and debugging.